packages/react/src/ReactElement.js

```diff
@@ -363,7 +363,11 @@ export function createElement(type, config, children) {
   if (type && type.defaultProps) {
     const defaultProps = type.defaultProps;
     for (propName in defaultProps) {
-      if (props[propName] === undefined) {
+      if (
+        props[propName] === undefined &&
+        propName !== 'ref' &&
```

Comment:
Is it actually an issue for users to define a default prop for refs? I understand why this is a bad practice for keys, since they should be unique. but would this be a bad practice for refs too?